### PR TITLE
Include "local only" JDKs

### PR DESCRIPTION
**Pull request by fbricon**

### CODE REVIEW

File: GenerateToolchainsFromSDKMAN.java [hunk 1]
```diff
@@ -135,7 +135,8 @@ private void run() {
                 continue;
             }
             // Find the locally installed JDK's
-            if (!"installed".equals(strings[4].trim())) {
+            String status = strings[4].trim();
+            if (!"installed".equals(status) && !"local only".equals(status)) {
```
Comment (tbee) [inline GenerateToolchainsFromSDKMAN.java:139]: I'm having trouble finding out what "local only" means. Why are local only not ok?

Comment (fbricon) [reply GenerateToolchainsFromSDKMAN.java:139]: When SDKMan has JDK installations that are no longer provided by SDKMan itself (eg. an "old" version of a JDK, that was previously installed by SDKMan), they appear as "local only". 
They're still OK to use. The code above skips everything that is not "installed" nor "local only", in other words, only accepts those 2 flavors.

Comment (tbee) [reply GenerateToolchainsFromSDKMAN.java:139]: local only is not installed on the path that sdkman installes its jdks, correct? 
~/.sdkman/candidates/java/...

**Review by tbee — APPROVED** [review]

File: GenerateToolchainsFromSDKMAN.java (hunk 1 above)
Comment (fbricon) [reply GenerateToolchainsFromSDKMAN.java:139]: See 
```
❯ ll ~/.sdkman/candidates/java
total 0
drwxr-xr-x  15 fbricon  staff   480B Oct  8 12:42 11.0.9-zulu
drwxr-xr-x  10 fbricon  staff   320B Oct 23 02:51 15.0.1.hs-adpt
drwxr-xr-x  10 fbricon  staff   320B Jan 21 13:04 15.0.2.hs-adpt
drwxr-xr-x  16 fbricon  staff   512B Jan 14 10:40 20.3.0.r8-grl
drwxr-xr-x  15 fbricon  staff   480B Feb  3 18:28 21.0.0.r11-grl
drwxr-xr-x  15 fbricon  staff   480B Feb  3 18:39 21.0.0.r8-grl
drwxr-xr-x  15 fbricon  staff   480B Oct 21 15:01 8.0.272.fx-librca
lrwxr-xr-x   1 fbricon  staff    14B Feb 25 13:44 current -> 15.0.2.hs-adpt
❯ sdk l java
================================================================================
Available Java Versions
================================================================================
 Vendor        | Use | Version      | Dist    | Status     | Identifier
--------------------------------------------------------------------------------
 AdoptOpenJDK  |     | 15.0.2.j9    | adpt    |            | 15.0.2.j9-adpt
               | >>> | 15.0.2.hs    | adpt    | installed  | 15.0.2.hs-adpt
               |     | 15.0.1.hs    | adpt    | local only | 15.0.1.hs-adpt
               |     | 11.0.10.j9   | adpt    |            | 11.0.10.j9-adpt
               |     | 11.0.10.hs   | adpt    |            | 11.0.10.hs-adpt
               |     | 8.0.282.j9   | adpt    |            | 8.0.282.j9-adpt
               |     | 8.0.282.hs   | adpt    |            | 8.0.282.hs-adpt
 Amazon        |     | 15.0.2.7.1   | amzn    |            | 15.0.2.7.1-amzn
               |     | 11.0.10.9.1  | amzn    |            | 11.0.10.9.1-amzn
               |     | 8.282.08.1   | amzn    |            | 8.282.08.1-amzn
 Azul Zulu     |     | 15.0.2       | zulu    |            | 15.0.2-zulu
               |     | 15.0.2.fx    | zulu    |            | 15.0.2.fx-zulu
               |     | 11.0.10      | zulu    |            | 11.0.10-zulu
               |     | 11.0.10.fx   | zulu    |            | 11.0.10.fx-zulu
               |     | 11.0.9       | zulu    | local only | 11.0.9-zulu
               |     | 8.0.282      | zulu    |            | 8.0.282-zulu
               |     | 8.0.282.fx   | zulu    |            | 8.0.282.fx-zulu
 BellSoft      |     | 15.0.2.fx    | librca  |            | 15.0.2.fx-librca
               |     | 15.0.2       | librca  |            | 15.0.2-librca
               |     | 11.0.10.fx   | librca  |            | 11.0.10.fx-librca
               |     | 11.0.10      | librca  |            | 11.0.10-librca
               |     | 8.0.282.fx   | librca  |            | 8.0.282.fx-librca
               |     | 8.0.282      | librca  |            | 8.0.282-librca
               |     | 8.0.272.fx   | librca  | local only | 8.0.272.fx-librca
 GraalVM       |     | 21.0.0.r11   | grl     | local only | 21.0.0.r11-grl
               |     | 21.0.0.r8    | grl     | local only | 21.0.0.r8-grl
               |     | 21.0.0.2.r11 | grl     |            | 21.0.0.2.r11-grl
               |     | 21.0.0.2.r8  | grl     |            | 21.0.0.2.r8-grl
               |     | 20.3.1.2.r11 | grl     |            | 20.3.1.2.r11-grl
               |     | 20.3.1.2.r8  | grl     |            | 20.3.1.2.r8-grl
               |     | 20.3.0.r8    | grl     | local only | 20.3.0.r8-grl
               |     | 19.3.5.r11   | grl     |            | 19.3.5.r11-grl
               |     | 19.3.5.r8    | grl     |            | 19.3.5.r8-grl
 Java.net      |     | 17.ea.12     | open    |            | 17.ea.12-open
               |     | 17.ea.11     | open    |            | 17.ea.11-open
               |     | 17.ea.10     | open    |            | 17.ea.10-open
               |     | 17.ea.9      | open    |            | 17.ea.9-open
               |     | 17.ea.2.pma  | open    |            | 17.ea.2.pma-open
               |     | 17.ea.2.lm   | open    |            | 17.ea.2.lm-open
               |     | 16.ea.36     | open    |            | 16.ea.36-open
               |     | 15.0.2       | open    |            | 15.0.2-open
               |     | 11.0.2       | open    |            | 11.0.2-open
 SAP           |     | 15.0.2       | sapmchn |            | 15.0.2-sapmchn
               |     | 11.0.10      | sapmchn |            | 11.0.10-sapmchn
 TravaOpenJDK  |     | 11.0.9       | trava   |            | 11.0.9-trava
               |     | 8.0.232      | trava   |            | 8.0.232-trava
================================================================================
Use the Identifier for installation:

    $ sdk install java 11.0.3.hs-adpt
================================================================================
```

Comment (tbee) [reply GenerateToolchainsFromSDKMAN.java:139]: Indeed! I tested it. Very good PR. Thank you.

**Pull request merged**